README.md

```diff
@@ -43,6 +44,8 @@ console.log(ambiguousCards[2].niceType); // 'Maestro'
 
 If no card types are found, this returns an empty array.
 
+*Note:* The card number provided should be normalized ahead of time. The card number string should not contain any non-integer values (e.g. no letters or special characters)
```

Comment:
```suggestion
*Note:* The card number provided should be normalized ahead of time. The card number string should not contain any non-integer values (e.g. no letters or special characters)

A simple way to do this:

```js
function normalizeCardNumber(cardNumber) {
  return String(cardNumber).replace(/[^\d]/g, "");
}
```
```

Reply:
While this _is_ helpful, I hesitate to add it just because our docs state we aren't a validation lib here, so documenting how to also seems unnecessary.